src/Service.Tests/dab-config.MsSql.json

```diff
@@ -2478,6 +2478,39 @@
         "bkname": "name"
       }
     },
+    "BooksSold": {
+      "source": {
+        "object": "books_sold",
+        "type": "table"
+      },
+      "graphql": {
+        "enabled": true,
+        "type": {
+          "singular": "books_sold",
+          "plural": "books_solds"
+        }
+      },
+      "rest": {
+        "enabled": true,
+        "methods": [
+          "get",
+          "post",
+          "put",
+          "patch",
+          "delete"
+        ]
```

Comment:
without this, does startup fail? I know Shyam has a PR out to fix this, but if it doesn't break, you should not include the methods section for a table.